packages/remix-dev/config.ts

```diff
@@ -141,7 +155,10 @@ export interface RemixConfig {
    */
   devServerBroadcastDelay: number;
 
-  mdx?: RemixMdxConfig | RemixMdxConfigFunction;
+  /**
+   * Configuration that effects how the bundles are generated.
+   */
+  buildOptions: RemixBuildOptions;
```

Comment:
How is `buildOptions` distinguished from other remix config options in your mind? Can we come up with a better name for what you're looking for? Or just leave the config flat without adding another key?

Reply:
The distinguishing feature to me of these properties are that they effect the bundled output contents / format, rather than where it ends up.

Reply:
Maybe compilerOptions?